src/components/Web3ReactManager/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
I don't know what you guys think (@aliangm @hachiojidev ) but for me , this component and the `useEagerConnect` should be merged together. The logic in this component seems to be a continuation of the `useEagerConnect` hook, and I don't see a use case where we only want to use `useEagerConnect`.

Also, it seems that this component render a loaders while trying to connect, we probably want to render the website even if the user is not connected....